bears/python/requirements/PySafetyBear.py

```diff
@@ -74,12 +74,20 @@ def run(self, filename, file):
                 version_spec_match.end(1) + 1,
             )
 
-            yield Result(
-                self,
-                message_template.format(vuln=vulnerability),
-                additional_info=vulnerability.description,
```

Comment:
Only the  `vulnerability.description` fetch should be within a `try:` block ; not the entire `yield`.

And `description` didnt disappear ; it was renamed.

Reply:
What changes you want for your second part of `review comment` or i will take ist as a suggestion

Reply:
Did they renamed to `advisory`

Reply:
Done  changes 👍